plutus-core/untyped-plutus-core/testlib/Evaluation/Builtins/Definition.hs

```diff
@@ -652,6 +654,8 @@ test_ExpModInteger = testNestedM "ExpMod" $ do
     evals @Integer 0 b [] [int 500, int (-1777), one] -- base:*, exp: neg, mod:1
     fails "mod-zero" b [] [one, one, zero] -- base:*, exp:*, mod: 0
     fails "mod-neg" b [] [one, one, int (-3)] -- base:*, exp:*, mod: neg
+    -- base is zero, negative exponent
+    fails "exp-neg-non-inverse0" b [] [int 0, int (-1), int 7]
```

Comment:
This is the case where the bug gave the worng result: none of the previous tests detected that.